public/app/plugins/panel/xychart/TooltipView.tsx

```diff
@@ -102,7 +107,7 @@ export const TooltipView = ({
     name: getFieldDisplayName(yField, frame),
     val: yField.values[rowIndex],
     field: yField,
-    color: series.pointColor(frame) as string,
```

Comment:
will the color from `series.pointColor` ever be different from `colorFacet`? It seems like there might be an unhandled case where colorFacet is undefined and color will always default to `#0000`

Reply:
hmm, apparently it can return a string _or_ an array of strings :exploding_head:

https://github.com/grafana/grafana/blob/701c6b6f074d4bc515f0824ed4de1997db035b69/public/app/plugins/panel/xychart/scatter.ts#L125-L134

>      // Yes we can improve this later 

(21 months ago) :rofl:

Reply:
Note -- this updates the logic so we do not try to re-calculate the color.  Instead it uses the value that was created previously